src/styles/_header.scss

```diff
@@ -0,0 +1,54 @@
+.header {
+  padding: 10px 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  &_link_logo {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  &_item:not(:last-child) {
+    margin-right: 20px;
+  }
+}
+
+.nav_link {
+  font-size: 12px;
+  text-transform: uppercase;
+  color: $mainColor;
+  font-weight: 500;
+  font-style: normal;
+  text-decoration: none;
+  transition: color 300ms ease;
+
+  &:hover {
+    color: $accentColor;
+  }
+
+  &--active {
+    color: $accentColor;
+    position: relative;
+
+    &::after {
+      content: '';
+      width: 100%;
+      height: 4px;
+      border-radius: 8px;
+      display: block;
+      position: absolute;
+      bottom: -19px;
+      left: 0;
+      right: 0;
+      background-color: $accentColor;
```

Comment:
The variable `$accentColor` is used here, but it is not defined within this file. Ensure that it is imported or defined elsewhere in your SCSS files.